libs/utils/android/system.py

```diff
@@ -63,9 +76,9 @@ def systrace_start(env, trace_file, time=None,
                 return None
 
         #  Format the command according to the specified arguments
-        systrace_pattern = "{} -e {} -o {} {}"
+        systrace_pattern = "{} -e {} -o {} -b {} {}"
```

Comment:
Without specifying the buffer size argument, systrace wouldn't record more than ~1 second of intensive workload. It did seem to work by specifying a bigger buffer size, so I suppose it does work ?